content/aws-account-creation/_index.md

```diff
@@ -6,6 +6,115 @@ weight: 100
 ---
 
 ## AWS Account Creation Steps
-This section is meant to serve as a guide to create a new AWS account using all of the free options. 
+The following information  is meant to serve as a guide to create a new AWS account using all of the free options. 
 
-I also want to potentially include an introduction to the billing dashboard so that learners can ensure they are not spending more money than intended for any given payment period.
+{{% notice note %}}
+Please note that you will need to provide a valid credit card while creating your AWS Account. You will not be immediately charged with any fees. This course will utilize all available free-tiers on the platform so that you accrue the absolute minimum ammount possible.
```

Comment:
```suggestion
Please note that you will need to provide a valid credit card while creating your AWS Account. You will not be immediately charged with any fees. This course will utilize all available free-tiers on the platform so that you accrue the absolute minimum amount possible.
```